src/nxdoc/nuxeo-server/monitoring-and-maintenance/hotfixes-installation-notes-for-nuxeo-platform-lts-2016.md

```diff
@@ -133,12 +133,18 @@ If you have any questions, feel free to contact our support team via a dedicated
 
 ### Security Issue on MS Windows
 
-Windows users need to upgrade the Tomcat version to protect against a potential CSRF token leak in the default manager applications distributed with Tomcat. Follow this procedure to upgrade Tomcat to version `7.0.81`, version recommended by [NXP-19726](https://jira.nuxeo.com/browse/NXP-19726):
-1. Download the PowerShell script [upgrade_tomcat7.ps1](https://github.com/nuxeo/nuxeo/blob/8.10/nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/upgrade_tomcat7.ps1)
-2. In a PowerShell session, run the following command where NUXEO_HOME is the path to your Nuxeo instance
+Windows users need to upgrade the Tomcat version to protect against a potential CSRF token leak in the default manager applications distributed with Tomcat. Follow this procedure to upgrade Tomcat to version `7.0.81` (or above), version recommended by [NXP-19726](https://jira.nuxeo.com/browse/NXP-19726):
```

Comment:
or above -> this should not be written because the hotfix mechanism is expecting 7.0.81 version of Tomcat. Of course customer can upgrade to a later version but he/she has then to modify manually some files.